code/modules/resleeving/machines.dm

```diff
@@ -526,9 +526,10 @@
 
 	//Re-supply a NIF if one was backed up with them.
 	if(MR.nif_path)
-		var/obj/item/device/nif/nif = new MR.nif_path(occupant,MR.nif_durability)
+		var/obj/item/device/nif/nif = new MR.nif_path(occupant,,MR.nif_savedata)
```

Comment:
Just use named parameters instead of this spooky double-comma that people might miss.